scripts/feedback.js

```diff
@@ -0,0 +1,16 @@
+import { invalid } from "../utils.js";
+import { apiClient } from "./utils/apiClient.js";
+
+export const sendFeedback = async (message) => {
+  if (!message) {
+    return;
+  }
+  const response = await apiClient.post({
+    path: "/v2/cli-feedback/",
+    body: { message }
+  });
+  if (!response.ok) {
+    return invalid("Unable to send feedback at this time, please try again.");
+  }
```

Comment:
If the  response is not ok and the `response.status` is 401 (Unauthorized), Can you return a different message? basically tell the user to login first. I think maybe something like `Please, run the 'login' command first to authenticate, then try again.`

You can also improve wording if you wish.